codeforboston/vaccinatema
# Rip out backend and dependencies.

This PR removes the backend of VaccinateMA! The public pages should still be entirely functional (though they use static, stale data thanks to #168 and #169).

## Test Plan:
* Delete and reinstall node modules. (`rm -rf node_modules && npm install`)
* Run `npm start` and navigate to localhost:3000.
* Ensure all pages load as usual and without console errors!
